easybuild/easyblocks/generic/mesonninja.py

```diff
@@ -51,6 +53,9 @@ def extra_options(extra_vars=None):
         extra_vars.update({
             'configure_cmd': [DEFAULT_CONFIGURE_CMD, "Configure command to use", CUSTOM],
             'separate_build_dir': [True, "Perform build in a separate directory", CUSTOM],
+            'build_cmd': [DEFAULT_BUILD_CMD, "Build command to use", CUSTOM],
+            'builddir': [None, "builddir to pass to meson", CUSTOM],
```

Comment:
For consistency, wouldn't `build_dir` be better here?

```suggestion
            'build_dir': [None, "Build directory to pass to meson", CUSTOM],
```

Reply:
for consistency with... `installdir` ?

Reply:
For consistency with `install_cmd` below. ;)

There's `self.installdir` and `self.builddir`, sure, but those are not easyconfig parameters.

Reply:
so, I shall also update existing code that is not being modified by this PR (i.e. lines 75-77)?